Manage Logon Hour via Windows PowerShell

```diff
@@ -0,0 +1,345 @@
+---
+post_title: Manage Logon Hour via Windows PowerShell
+username: farisnt@gmail.com
+Catagories: PowerShell, Active Directory
+Summary: How to change users Logon Hours using PowerShell
+---
+
+
+**Q:** How can I set AD domain user Logon Hour via Windows PowerShell ?
+
+**A:** Most of the Active Directory object properties can be easily read or modified by using the `Get-` or `Set-` with the property name and its value, but when it comes to **LogonHours**, it not that easy. There are multiple steps and challenges required to build the value, and in this tutorial, you will learn all the tips and tricks needed to set the **LogonHours** hours via Windows PowerShell. So get a cup of coffee, and let's start.
```

Comment:
please recase this article with semantic sentences.
Each Sentence in a paragraph should be  on a separate line. 
This make it easy to comment on the code/article